src/index.ts

```diff
@@ -41,6 +42,7 @@ export class HLTVFactory {
   getTeam = getTeam(this.config)
   getTeamStats = getTeamStats(this.config)
   getPlayer = getPlayer(this.config)
+  getPlayerByName = getPlayerByName(this.config)
```

Comment:
Maybe we can add a different option to `getPlayer` instead? Currently only `getPlayer({ id: 1234 })` works, but we can add a name parameter so that `getPlayer({ name: 'someone' })` is possible. What do you think?

Reply:
Yes it would be easier.

Reply:
I played a bit around but i don't know how too change this in this setup.